recipes/libbasisu/all/conanfile.py

```diff
@@ -0,0 +1,104 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+
+class LibBasisUniversalConan(ConanFile):
+    name = "libbasisu"
+    description = "Basis Universal Supercompressed GPU Texture Codec"
+    homepage = "https://github.com/BinomialLLC/basis_universal"
+    topics = ("conan", "basis", "textures", "compression")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "Apache-2.0"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "fPIC": [True, False],
+        "shared": [True, False],
+        "sse": [True, False],
```

Comment:
What level of sse support is this?
x86_64 demands a minimum level of sse support.
Is this for x86 support? Or is this for x86_64 with extra sse? e.g. SSE4?

Reply:
It's support for SSE4.1, I renamed it to `use_sse4`.